programs/sample_enroll/sample_enroll.c

```diff
@@ -75,6 +75,12 @@ struct enroll_cert {
 #include <ziti/zitilib.h>
 
 int main(int argc, char **argv) {
+
+    if (argc != 2) {
```

Comment:
is this from the other PR?

Reply:
Oops, sorry. I accidentally push to this branch also..

Reply:
Redundant commit is removed.